runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -396,12 +457,12 @@ class TR_VectorAPIExpansion : public TR::Optimization
    *   \param id
    *     Symbol reference for which all transitive aliases need to be found
    */
```

Comment:
Please add descriptions for the new parameters. `aliasesfield` and `classField`.